brave/browser-laptop
# Load search favicons locally in about:preferences

fix https://github.com/brave/browser-laptop/issues/12689
fix https://github.com/brave/browser-laptop/issues/11108
partial fix for https://github.com/brave/browser-laptop/issues/12632

Test Plan:
1. go to about:preferences and open the 'network requests' tab in devtools
2. click on the search preferences tab. you should not see any network requests to remote sites.
3. all the search favicons should load correctly

Submitter Checklist:


Test Plan:


Reviewer Checklist:


Tests
